Dockerfile

```diff
@@ -7,35 +7,38 @@ MAINTAINER Tim Sutton<tim@kartoza.com>
 RUN apt-get -y update && \
     apt-get install -y \
     gettext \
-    python-yaml \
+    python3-yaml \
     libgeos-dev \
-    python-lxml \
+    python3-lxml \
     libgdal-dev \
     build-essential \
     python-dev \
     libjpeg-dev \
     zlib1g-dev \
     libfreetype6-dev \
-    python-virtualenv
+    python3-virtualenv
+RUN pip install Shapely Pillow MapProxy uwsgi pyproj
 
-RUN pip install Shapely Pillow MapProxy uwsgi
+RUN set -eux; \
```

Comment:
It would be best if these apt commands are added with the one above. The resulting image will be cleaner